components/footer/src/test.tsx

```diff
@@ -154,4 +170,12 @@ describe('Footer', () => {
   it('matches with navigation and meta snapshot', () => {
     expect(mount(<FooterWithNavigationAndMeta />)).toMatchSnapshot('FooterWithNavigationAndMeta');
   });
+
+  it('matches with custom licence', () => {
+    expect(mount(<FooterWithCustomLicence />)).toMatchSnapshot('FooterWithCustomLicence');
+  });
```

Comment:
for what it's worth, no need to add any new snapshot tests in as we want to move away from them, as long as they're covered by visual regression (i.e. stories in storybook that are picked up by Chromatic)